lib/product_loader.rb

```diff
@@ -0,0 +1,14 @@
+require 'json'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.